src/libstd/sys/unix/process/process_fuchsia.rs

```diff
@@ -164,6 +164,36 @@ impl Process {
         }
         Ok(ExitStatus::new(proc_info.rec.return_code))
     }
+
+    pub fn try_wait(&mut self) -> io::Result<ExitStatus> {
+        use default::Default;
+        use sys::process::magenta::*;
+
+        let mut proc_info: mx_info_process_t = Default::default();
+        let mut actual: mx_size_t = 0;
+        let mut avail: mx_size_t = 0;
+
+        unsafe {
+            let status = mx_handle_wait_one(self.handle.raw(), MX_TASK_TERMINATED,
+                                            0, ptr::null_mut());
+            match status {
+                0 => { }, // Success
+                x if x == ERR_TIMED_OUT => {
```

Comment:
I believe you can use constants directly as patterns, so this arm could be written as just `ERR_TIMED_OUT => {` with no `x if x ==`.